libraries/pushproviders/unifiedpush/build.gradle.kts

```diff
@@ -42,7 +42,13 @@ dependencies {
     implementation(libs.serialization.json)
 
     // UnifiedPush library
-    api(libs.unifiedpush)
+    implementation(libs.unifiedpush) {
+        // Exclude package com.google.crypto.tink
+        // Duplicate classes between
+        // tink-1.16.0.jar -> tink-1.16.0 (com.google.crypto.tink:tink:1.16.0)
+        // tink-android-1.8.0.jar tink-android-1.8.0 (com.google.crypto.tink:tink-android:1.8.0)
+        exclude(group = "com.google.crypto.tink", module = "tink")
```

Comment:
I think this has to be removed now the dependency with the transitivity dependency to tink is gone.

Reply:
Done in [57283a8](https://github.com/element-hq/element-x-android/pull/4358/commits/57283a89c2ac878c53183cb75a1546218cee80a3)

Reply:
it seems to be still necessary

Reply:
Weird, it wouldn't let me build the app locally with this exclusion 🫤 .